bioptim/interfaces/ipopt_interface.py

```diff
@@ -84,7 +85,7 @@ def online_optim(self, ocp, show_options: dict = None):
             raise RuntimeError("Online graphics are not available on Windows")
         self.options_common["iteration_callback"] = OnlineCallback(ocp, show_options=show_options)
 
-    def configure(self, solver_options: dict):
+    def configure(self, solver_options: SolverOptionsIPOPT):
         """
         Set some Ipopt options
```

Comment:
This method is probably not required anymore but a `to_dict` method should be added to `SolverOptions` as an abstractmethod that returns a valid options dictionary depending on the Options